2024Robot_Team3484/src/main/cpp/subsystems/IntakeSubsystem.cpp

```diff
@@ -0,0 +1,115 @@
+//
+// Intake Subsystem
+//
+// -Noah, Aidan  & Ethan
+//
+
+#include <subsystems/IntakeSubsystem.h>
+#include <units/angle.h>
+
+using namespace IntakeConstants;
+
+IntakeSubsystem::IntakeSubsystem(
+    int PIVOT_MOTOR_CAN_ID, 
+    int DRIVE_MOTOR_CAN_ID, 
+    int PIECE_SENSOR_DI_CH, 
+    int ARM_SENSOR_DI_CH,
+
```

Comment:
Confirm naming convention on parameters with the rest of the team. I know constants are typed in all caps, but I don't believe function/constructor parameters are. The concern I have here is the all-caps designation would lead one to believe this is a constant not a parameter.